This check fails on cross, it produces false on host, so we just skip it
entirely.

--- configure	2020-12-28 15:47:57.495364388 +1100
+++ configure	2020-12-28 15:48:45.881677930 +1100
@@ -36944,71 +36944,6 @@
 fi
 
 
-echo "$as_me:$LINENO: checking whether printf supports %p" >&5
-echo $ECHO_N "checking whether printf supports %p... $ECHO_C" >&6
-if test "${cvs_cv_func_printf_ptr+set}" = set; then
-  echo $ECHO_N "(cached) $ECHO_C" >&6
-else
-  if test "$cross_compiling" = yes; then
-  { { echo "$as_me:$LINENO: error: cannot run test program while cross compiling
-See \`config.log' for more details." >&5
-echo "$as_me: error: cannot run test program while cross compiling
-See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
-else
-  cat >conftest.$ac_ext <<_ACEOF
-/* confdefs.h.  */
-_ACEOF
-cat confdefs.h >>conftest.$ac_ext
-cat >>conftest.$ac_ext <<_ACEOF
-/* end confdefs.h.  */
-#include <stdio.h>
-/* If printf supports %p, exit 0. */
-int
-main ()
-{
-  void *p1, *p2;
-  char buf[256];
-  p1 = &p1; p2 = &p2;
-  sprintf(buf, "%p", p1);
-  exit(sscanf(buf, "%p", &p2) != 1 || p2 != p1);
-}
-_ACEOF
-rm -f conftest$ac_exeext
-if { (eval echo "$as_me:$LINENO: \"$ac_link\"") >&5
-  (eval $ac_link) 2>&5
-  ac_status=$?
-  echo "$as_me:$LINENO: \$? = $ac_status" >&5
-  (exit $ac_status); } && { ac_try='./conftest$ac_exeext'
-  { (eval echo "$as_me:$LINENO: \"$ac_try\"") >&5
-  (eval $ac_try) 2>&5
-  ac_status=$?
-  echo "$as_me:$LINENO: \$? = $ac_status" >&5
-  (exit $ac_status); }; }; then
-  cvs_cv_func_printf_ptr=yes
-else
-  echo "$as_me: program exited with status $ac_status" >&5
-echo "$as_me: failed program was:" >&5
-sed 's/^/| /' conftest.$ac_ext >&5
-
-( exit $ac_status )
-cvs_cv_func_printf_ptr=no
-fi
-rm -f core *.core gmon.out bb.out conftest$ac_exeext conftest.$ac_objext conftest.$ac_ext
-fi
-rm -f core core.* *.core
-fi
-echo "$as_me:$LINENO: result: $cvs_cv_func_printf_ptr" >&5
-echo "${ECHO_T}$cvs_cv_func_printf_ptr" >&6
-if test $cvs_cv_func_printf_ptr = yes; then
-
-cat >>confdefs.h <<\_ACEOF
-#define HAVE_PRINTF_PTR 1
-_ACEOF
-
-fi
-
-
 # Try to find connect and gethostbyname.
 
 echo "$as_me:$LINENO: checking for main in -lnsl" >&5
